src/vscode-dts/vscode.proposed.badges.d.ts

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+declare module 'vscode' {
+
+	// https://github.com/microsoft/vscode/issues/62783 @matthewjamesadam
+
+	/**
+	 * Base class for badges
+	 */
+	export abstract class Badge {
+		/**
+		 * A label to present in tooltips
+		 */
+		readonly label: string;
+	}
+
+	/**
+	 * A badge presenting a number
+	 */
+	export class NumberBadge extends Badge {
```

Comment:
Let's delete the abstract class and just call this `Badge`.

Reply:
Done.  I also changed this to an `interface` instead of a `class`.  Without the object hierarchy the interface seemed like a better fit, and matched the rest of the API.